docs/additional-resources/videos.table.js

```diff
@@ -4,59 +4,187 @@ import React from "react";
+    author: "Mateusz Czerniawski",
+    date: "2019-06-05",
+    url: "https://www.youtube.com/watch?v=CMul62ef214",
+    event: "PowerShell Conference EU 2019",
+  },
+  {
+    title: "Breakout Session: Community Talk about Pester 5",
+    author: "Jakub Jareš",
+    date: "2019-06-05",
+    url: "https://www.youtube.com/watch?v=ysw_hXbYYs0",
+    event: "PowerShell Conference EU 2019",
+  },
   {
     title: "Custom Pester assertions are the vocabulary of your tests",
-    author: "Jakub Jares",
-    date: "2019-07-19",
+    author: "Jakub Jareš",
+    date: "2019-06-05",
     url: "https://www.youtube.com/watch?v=WYw23cGoKco",
+    event: "PowerShell Conference EU 2019",
+  },
+  {
+    title: "Pester internals and concepts",
+    author: "Jakub Jareš",
+    date: "2019-06-04",
+    url: "https://www.youtube.com/watch?v=6d0K2i7pgQs",
+    event: "PowerShell Conference EU 2019",
   },
   {
     title: "Beyond Pester 102: Acceptance testing with PowerShell",
     author: "Glenn Sarti",
     date: "2019-05-18",
     url: "https://www.youtube.com/watch?v=L-1nXtaQ6YM",
+    event: "PowerShell + DevOps Global Summit 2019",
   },
   {
     title: "Mastering PowerShell testing with Pester",
     author: "Mark Wragg",
     date: "2018-10-10",
     url: "https://www.youtube.com/watch?v=BbOiQCgDDR8",
+    event: "PSDay.UK 2018",
   },
   {
     title: "Beyond Pester 101: Applying testing principles to PowerShell",
     author: "Glenn Sarti",
     date: "2018-05-03",
     url: "https://www.youtube.com/watch?v=NrUxgSaFvtk",
+    event: "PowerShell + DevOps Global Summit 2018",
+  },
+  {
+    title: "Common mistakes in Pester tests",
+    author: "Jakub Jareš",
+    date: "2018-04-20",
+    url: "https://www.youtube.com/watch?v=VBSxCXmNqQ8",
+    event: "PowerShell Conference EU 2018",
+  },
+  {
+    title: "Infrastructure validation using Pester",
+    author: "Irwin Strachan",
+    date: "2018-04-18",
+    url: "https://www.youtube.com/watch?v=Qfi_H7IZyHg",
+    event: "PowerShell Conference EU 2018",
+  },
+  {
+    title: "Pester internals and concepts",
+    author: "Jakub Jareš",
+    date: "2018-04-18",
+    url: "https://www.youtube.com/watch?v=Wc-9B_MqxYs",
+    event: "PowerShell Conference EU 2018",
+  },
+  {
+    title: "How we made Configurable Pester Tests for SQL Server",
+    author: "Chrissy LeMaire, Rob Sewell",
+    date: "2018-04-17",
+    url: "https://www.youtube.com/watch?v=3XahgUEp12I",
+    event: "PowerShell Conference EU 2018",
+  },
+  {
+    title: "Fighting Configuration Drift with Dynamic Pester Tests",
+    author: "André Kamman",
+    date: "2017-05-04",
+    url: "https://www.youtube.com/watch?v=JCfwEpkjFfA",
+    event: "PowerShell Conference EU 2017",
+  },
+  {
+    title: "Green is bad Red is Good - Turning your Checklists into Pester Tests",
+    author: "Rob Sewell",
+    date: "2017-05-03",
+    url: "https://www.youtube.com/watch?v=Qy-uvT57pt8",
+    event: "PowerShell Conference EU 2017",
```

Comment:
Move to versioned_v4 docs?

Reply:
Agreed, but shouldn't this be until line 188 (the last video) or at least line 153 (the closing curly brace)?

Reply:
Yes, except any videos that are more "why test" or tips/tricks without much Pester code. 🙂

Planned to fix later, but feel free to include it here.

Reply:
I've reviewed the nine original videos. IMO "Custom Pester assertions are the vocabulary of your tests" (2019-07-19) from Jakub could remain as it addresses a specific topic and (at first sight) doesn't violate the v5 rules. The rest could be added to the versioned v4 docs (perhaps with the exception of the PluralSight one from Robert Cain, because that is behind a paywall).

I would also like to suggest to nonetheless add "Common mistakes in Pester tests" (2018-04-20) and "Pester internals and concepts" (2018-04-18) from Jakub. The first one because of the tips, the second one because the v5 foundation (before it was v5) is more or less explained.

Reply:
Sounds good. Did not expect you to go through all videos, so thanks! 😄

Reply:
I will try to adjust the PR tomorrow so that it will reflect what we've discussed.

Reply:
I've just added 5 more commits to this PR to reflect what we've discussed so far.